lighthouse-core/audits/seo/crawlable-anchors.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Audit = require('../audit.js');
+const i18n = require('../../lib/i18n/i18n.js');
+
+const UIStrings = {
+  /** Title of a Lighthouse audit that provides detail on whether anchors have hyperlinks which can be crawled by search engines. This descriptive title is shown when all hyperlinks on the page are crawlable. */
+  title: 'Anchors have crawlable hyperlinks',
```

Comment:
@AVGP I just want to double check that this is the appropriate SEO parlance to be using for this title.

It sounds a bit unusual to my understanding since I think of the anchor itself as being the hyperlink rather than the `href` attribute of the anchor being called the hyperlink, but I'm not that hip to the SEO lingo :)

Reply:
I'd reword that. SEO lingo is usually "links", not "anchors" and "href" not "hyperlinks" :)
So: "Lighthouse audit that provides detail on whether links have potentially-crawlable href attributes. The "potentially-crawlable" is the usual care it needs to make sure people don't derive a guarantee that we can/will crawl all href URLs.

Reply:
sounds like we agree we'd like to reword these slightly then.

based on @AVGP 's guidance, maybe just go super simple: `Links are crawlable`/`Links are not crawlable`?

Reply:
Thanks both, true that sounds more intuitive. Have attempted a reword: https://github.com/GoogleChrome/lighthouse/pull/10662/commits/d2095db1bd75423cc0eba3fb85472c5b27e5e14b

But let me know if you had something else in mind

Reply:
Hey @patrickhulce  even you left the comment before me GitHub didn't do the auto-updating comments thing it normally does 🤷‍♂️️ maybe because it's a large convo. ` Links are crawlable/Links are not crawlable` sounds good have updated the messages, along with my extra message changes, but happy to tweak those other messages further if needed.